Lombiq.Tests.UI/Models/InstanceCommandLineArgs.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+
+namespace Lombiq.Tests.UI.Models;
+
+public class InstanceCommandLineArgumentsBuilder
+{
+    private readonly List<string> _arguments = new();
+
+    public InstanceCommandLineArgumentsBuilder AddSwitch(string argument)
+    {
+        _arguments.Add($"{PrepareArg(argument)}");
+
+        return this;
+    }
+
+    public InstanceCommandLineArgumentsBuilder AddWithValue<T>(string key, T value)
+    {
+        _arguments.Add($"{PrepareArg(key)}={value}");
```

Comment:
Wrap the string interpolation with `FormattableString.Invariant()` because value may be an `IFormattable` such as `int`.